src/main/java/org/ibp/api/brapi/v1/germplasm/GermplasmAttributes.java

```diff
@@ -0,0 +1,28 @@
+package org.ibp.api.brapi.v1.germplasm;
+
+import org.generationcp.middleware.domain.germplasm.AttributeDTO;
+
+import java.util.List;
+
+public class GermplasmAttributes {
```

Comment:
add equals, hashCode and toString

Reply:
Ok

Reply:
@vmaletta Please add @AutoProperty annotation to this class, remember to add it always that you use pojomatics

Reply:
ok